Documentation/configuring-reporting-operator.md

```diff
@@ -129,10 +109,47 @@ In this example the externalIP of the LoadBalancer is `35.227.172.86` and the po
 curl "http://35.227.172.86:8080/api/v1/reports/get?name=cluster-memory-capacity-hourly&format=tab"
 ```
 
+### Openshift Authentication
+
+Authentication can be enabled by setting the options below to true.
+Enabling authentication configures the reporting-operator pod to run the Openshift auth-proxy as a sidecar container in the pod, adjusting the ports so that the reporting-operator API isn't exposed directly, but instead using the auth-proxy pod.
```

Comment:
Maybe break-up sentence into two. Something like:
```
... Openshift auth-proxy as a sidecar container in the pod. The sidecar adjusts the ports so that the reporting-operator API isn't exposed directly, but instead uses the auth-proxy pod.
```

Reply:
Good idea. I made an adjustment similar to your suggestion. How does it look now?

Reply:
Looks good now!